src/utils/time/index.js

```diff
@@ -20,24 +20,24 @@ export const calculateTime = (time, formatForChatTab = false) => {
   }
 
   if (minutes < 60) {
-    return minutes === 0 ? '1m ago' : `${minutes}m ago`;
+    return minutes === 0 ? '1m' : `${minutes}m`;
   }
 
   if (hours < 24) {
-    return `${hours}h ago`;
+    return `${hours}h`;
   }
 
   if (days === 1) {
-    return formatForChatTab ? `${days}d ago` : 'Yesterday';
+    return formatForChatTab ? `${days}d` : '1d';
   }
 
   if (days >= 2 && days <= 6) {
-    return `${days}d ago`;
+    return `${days}d`;
   }
 
   if (days >= 7 && days <= 13) {
-    return '1w ago';
+    return '1w';
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from '1w ago' to '1w' has the same issue as above - it loses the 'ago' context. Consider adding 'ago' back.
```diff
- return '1w';
+ return '1w ago';
```

42:
The change from `${weeks}w ago` to `${weeks}w` also loses the 'ago' context. Consider adding 'ago' back.
```diff
- return `${weeks}w`;
+ return `${weeks}w ago`;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->